### ISSUE TITLE
DLITE loss function evaluation results comparisions

### ISSUE THREAD

**Issue by lixiao-yang** [keweimao/DeepDelight#3]
Hi all, according to my version of the DLITE loss function replacing the cross entropy loss. The results show a much lower evaluation loss for the Shake corpus. I tried to change the dropout rate but no significant changes. If you are interested, please take a look at the comparision.ipynb file at #2 and the DLITE loss function implementation in DLITE.ipynb in #2 .

**Comment by keweimao** [comment]
Both points are very interesting! Let's discuss them further and hopefully experiment further to verify the observations here.

**Comment by lixiao-yang** [comment]
I used the modified DLITE functions but the results show a very large loss compared to cross-entropy, which might needs more examine: [Updated DLITE](https://github.com/keweimao/DeepDelight/blob/main/Lixiao/DLITE%20loss%20function/DLITE_loss.ipynb)

**Comment by keweimao** [comment]
DLITE scores are always in the [0, 1] range. 

It's likely a bug/incorrect formula in the DLITE implementation causing the large loss scores.

**Comment by lixiao-yang** [comment]
DLITE loss value range problem fixed, issue closed.
